lib/trento/abilities/users_abilities.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Trento.Abilities.UsersAbilities do
+  @moduledoc """
+  Many to main table schema used to connect users and abilities.
```

Comment:
`Many to main` -> `Many to many`.

And the next maybe better:
`We have a dedicated schema to implement the association between users and abilities, to just enable read only operation. Using the default ecto schema, declaring a user with a not existing ability would trigger the creation of this second, and we don't want to allow that. Abilities are just read only.

Reply:
done